File: 3. DSA/04. Python/01. Basics/Question10.py
# Factorial of a given integer

n = int(input("Enter the number: "))
factorial = 1

for i in range(1, n + 1):
    factorial *= i

print(f"The factorial of {n} is {factorial}")
# print("The factorial of {} is {}".format(n, factorial))
# print("The factorial of ", n, " is ", factorial, sep = '')

File: 3. DSA/02. C++/01. Basics/Question10.cpp
// Factorial of a given integer

#include <iostream>
using namespace std;

int main()
{
    int num;
    cout << "Enter a number: ";
    cin >> num;

    int fact = 1;
    for(int i = num; i >= 1; i--)
    {
        fact *= i;
    }

    if(num < 0)
        fact = 0;

    cout << "\nThe factorial of " << num << " is " << fact << endl;
}